src/tangle.ts

```diff
@@ -230,14 +283,18 @@ export class Client<T> {
                     if (event.transient) {
                         return { transient: { ...transient, ...event.transient } };
                     }
-
-                    throw new Error('Neither event nor state change was given');
+                    // throw new Error('Neither event nor state change was given');
```

Comment:
@christian-bromann do you remember what this was for? I replace it with L289-292 to don't emit any "empty" values. I'd like to remove the commented-out line.

Reply:
> I replace it with L289-292

Do you have a link?

> I'd like to remove the commented-out line.

Did it bug you while implementing this PR? Afaik this should never happen so IMO it is worth keeping it to ensure we are aware when it happens nonetheless

Reply:
It actually will happen now because context can be the sole property set in the payload.

For reference this what I added to account for empty payloads: https://github.com/stateful/tangle/pull/32/files/1bb55ccc6c8ae360fd5de33ae8b885f75406e4a8#diff-d69a27fb88eae301e3d525c17615414ef68d4fba199147e374adc4372ab23c05R289-R292

Reply:
LGTM 👍